package.json

```diff
@@ -8,8 +8,8 @@
     "scripts": {
         "clean": "shx rm -rf lib 0x_mesh/db",
         "build": "tsc -p tsconfig.json",
-        "test": "ETHEREUM_RPC_URL=http://localhost:8545 CHAIN_ID=1337 mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --timeout 100000 --exit",
-        "dev": "nodemon -r dotenv/config src/index.ts | pino-pretty",
+        "test": "ETHEREUM_RPC_URL=http://localhost:8545 CHAIN_ID=1337 RFQT_API_KEY_WHITELIST='[\"koolApiKey1\"]' RFQT_MAKER_ENDPOINTS='[\"http://localhost:4000\"]' mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --timeout 100000 --exit",
+        "dev": "ETHEREUM_RPC_URL=http://localhost:8545 CHAIN_ID=1337 RFQT_API_KEY_WHITELIST='[\"koolApiKey1\"]' RFQT_MAKER_ENDPOINTS='[\"http://localhost:4000\"]' nodemon -r dotenv/config src/index.ts | pino-pretty",
```

Comment:
Is this used anywhere outside of a ganache dev environment?  Are breaking other use cases by introducing the ganache RPC URL here?

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/d11b8508ead8586d348f94d154460e13d453de1e